common/djangoapps/adaptive_learning/config/models.py

```diff
@@ -23,7 +23,7 @@ def feature_enabled(cls, course_id=None):
         If the course_id parameter is passed, it returns a boolean indicating
         whether Adaptive Learning is available for the particular course.
         Note that in order for Adaptive Learning to be available in a given course
-        both flags (an instance-wide and and a course-specific ones)
+        both flags (an instance-wide flag and and a course-specific one)
```

Comment:
@tomaszgy This *still* contains a typo: "and" is duplicated. Simply copying the [string](https://github.com/open-craft/edx-platform/pull/72#discussion_r151661260) that I gave you would have helped ;)

Reply:
@itsjeyd, I'm sorry about yet another typo here. I should've copied it, indeed :)

Reply:
For the `isort` - yes, I did run it although I don't think it takes into account too much where the packages come from - it just made one section of all imports but `logging`, sorted alphabetically. But I'll move `adaptive_learning` part to the bottom.